usr/bin/whonix_firewall

```diff
@@ -13,13 +13,13 @@
 set -e
 
 if [ -f "/run/qubes/this-is-templatevm" ]; then
-   /usr/bin/whonix-workstation-firewall
+   whonix-workstation-firewall $@
```

Comment:
Pretty sure that needs to be quoted with `"$@"`.

https://mywiki.wooledge.org/WrapperScript

Reply:
682be4c